test/Dialect/Substrait/literal.mlir

```diff
@@ -1,6 +1,30 @@
 // RUN: substrait-opt -split-input-file %s \
 // RUN: | FileCheck %s
 
+// CHECK:      substrait.plan version 0 : 42 : 1 {
+// CHECK-NEXT:   relation
+// CHECK:         %[[V0:.*]] = named_table
+// CHECK-NEXT:    %[[V1:.*]] = project %[[V0]] : tuple<si1> -> tuple<si1, !substrait.binary> {
+// CHECK-NEXT:    ^[[BB0:.*]](%[[ARG0:.*]]: tuple<si1>):
+// CHECK-NEXT:      %[[V2:.*]] = literal "4,5,6,7" : !substrait.binary
+// CHECK-NEXT:      yield %[[V2]] : !substrait.binary
+// CHECK-NEXT:    }
+// CHECK-NEXT:    yield %[[V1]] : tuple<si1, !substrait.binary> 
+
+substrait.plan version 0 : 42 : 1 {
+  relation {
+    %0 = named_table @t1 as ["a"] : tuple<si1>
+    %1 = project %0 : tuple<si1> -> tuple<si1, !substrait.binary> {
+    ^bb0(%arg : tuple<si1>):
+      %bytes = literal "\34\2c\35\2c\36\2c\37" : !substrait.binary
```

Comment:
This also works:

```suggestion
      %bytes = literal "4,5,6,7" : !substrait.binary
```

Is there a reason not to do it this way but to escape instead?

Reply:
Nit: maybe use some other content? This might be read as an array of the bytes `4`, `5`, `6`, and `7`.